tests/test_websockets.py

```diff
@@ -522,3 +628,22 @@ async def app(scope: Scope, receive: Receive, send: Send):
     with pytest.raises(RuntimeError):
         with client.websocket_connect("/") as websocket:
             websocket.send({"type": "websocket.connect"})
+
+
+def test_send_disconnect_no_code(test_client_factory: Callable[..., TestClient]):
+    """Test that a client close message with a missing status code is accepted,
+    and verify the message passed to the application."""
+
+    close_msg: Message = {}
+
+    async def app(scope: Scope, receive: Receive, send: Send) -> None:
+        nonlocal close_msg
+        websocket = WebSocket(scope, receive=receive, send=send)
+        await websocket.accept()
+        close_msg = await websocket.receive()
+
+    client = test_client_factory(app)
+    with client.websocket_connect("/") as websocket:
+        websocket.send({"type": "websocket.disconnect"})
+
+    assert close_msg == {"type": "websocket.disconnect"}
```

Comment:
@kristjanvalur This test doesn't make sense to me. Also, this is more a test for the `TestClient` than the `WebSocket`, so it should go in `test_testclient.py`.

I'll remove it, but if you want, feel free to open a new PR adding the code on the `TestClient`.

Reply:
sure, I'll remove it.